firefox-ios/Client/Frontend/Library/Bookmarks/BookmarksViewController.swift

```diff
@@ -647,7 +595,12 @@ extension BookmarksViewController: Notifiable {
 extension BookmarksViewController {
     func bottomLeftButtonAction() {
         if state == .bookmarks(state: .inFolderEditMode) {
-            presentInFolderActions()
+            guard let bookmarkFolder = self.viewModel.bookmarkFolder else { return }
+
+            self.bookmarkCoordinatorDelegate?.showBookmarkDetail(
```

Comment:
I think we can remove `self.` usage here
```suggestion
            guard let bookmarkFolder = viewModel.bookmarkFolder else { return }

            bookmarkCoordinatorDelegate?.showBookmarkDetail(
```